slides/src/content/original/routing.md

```diff
@@ -0,0 +1,297 @@
+# Routing
+
+---
+
+## Why Routing?
+
+- Routing allows the user to navigate to different parts of the application.
+- It keeps your application linkable and bookmarkable and allow users to share links with others.
```

Comment:
Not sure if this is a thing, but these list items are now sentences with periods. Up until now, list items have been less formal sentence fragments. You could DRY these up with:

Why Routing?
* Allows the user to navigate to different parts of the application
* Keeps your application linkable and bookmarkable...

Routing allows you to:
* Maintain the state of the application
* Create modular applications
...
for the remainder of the page.

Note the lack of periods.